packages/monorepo-scripts/src/utils/doc_generate_and_upload_utils.ts

```diff
@@ -103,6 +103,9 @@ export class DocGenerateAndUploadUtils {
             switch (node.kind) {
                 case ts.SyntaxKind.ExportDeclaration: {
                     const exportClause = (node as any).exportClause;
+                    if (_.isUndefined(exportClause)) {
```

Comment:
@fabioberger commenting here to make sure you get a chance to review the changes I made to this file. It's easy to miss in a PR of this size.